ShenkinStore/Models/Product.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace ShenkinStore.Models
+{
+    public class Product
```

Comment:
There are no validations on this please add some for the important fields

Reply:
There are some auto validations for the fields.
Do you want me to change it anyway to our validations?

Reply:
It will trigger errors on the UI for missing\incorrect values?